classroom/snippets/snippets.py

```diff
@@ -13,13 +13,110 @@
 # limitations under the License.
 
 from __future__ import print_function
+import json
 from googleapiclient import errors
 
 
 class ClassroomSnippets(object):
     def __init__(self, service):
         self.service = service
 
+    def create_course(self):
+        """
+        Creates a 10th Grade Biology course.
+        """
+        service = self.service
+        # [START classroom_create_course]
+        course = {
+            'name': '10th Grade Biology',
+            'section': 'Period 2',
+            'descriptionHeading': 'Welcome to 10th Grade Biology',
+            'description': """We'll be learning about about the
+                         structure of living creatures from a
+                         combination of textbooks, guest lectures,
+                         and lab work. Expect to be excited!""",
+            'room': '301',
+            'ownerId': 'me',
+            'courseState': 'PROVISIONED'
+        }
+        course = service.courses().create(body=course).execute()
+        print('Course created:', course.get('name'), course.get('id'))
+        # [END classroom_create_course]
+
+    def get_course(self):
+        """
+        Retrieves a classroom course by its id.
+        """
+        service = self.service
+        # [START classroom_get_course]
+        course_id = '123456'
+        try:
+            course = service.courses().get(id=course_id).execute()
+            print('Course "{%s}" found.', course.get('name'))
+        except errors.HttpError as e:
+            error = json.loads(e.content).get('error')
```

Comment:
We don't need to be this specific for the snippet. I'd return the response and end the snippet before the return.